svelte/src/routes/v2/scripts/draw/tick.ts

```diff
@@ -1,63 +1,35 @@
 const PADDING = 10;
+
 export function innerTicked(membersContainerElement: any, memberElements: any) {
 	membersContainerElement.attr('transform', (d: any) => `translate(${d.x},${d.y})`);
 	memberElements.attr('width', (d: any) => d.width).attr('height', (d: any) => d.height);
 	memberElements.attr('x', (d: any) => d.cx).attr('y', (d: any) => d.cy);
 }
 export function linkTicked(linkElements: any) {
-	linkElements
-		.attr('x1', (d: any) => {
-			// change to global coordinates.
-			let result = d.source.x;
-
-			let temp = d.source;
-			while (temp.parent) {
-				result += temp.parent.x;
-				temp = temp.parent;
-			}
-
-			return result;
-		})
-		.attr('y1', (d: any) => {
-			// change to global coordinates.
-			let result = d.source.y;
-
-			let temp = d.source;
-			while (temp.parent) {
-				result += temp.parent.y;
-				temp = temp.parent;
-			}
+	linkElements.attr('d', function (this: SVGPathElement, d: any) {
```

Comment:
this part will conflict with my next PR. I think I am gonna need your help to merge them. I hope that's okay with you